test/v2/address.js

```diff
@@ -170,12 +190,13 @@ describe("#AddressRouter", () => {
       // Mock the Insight URL for unit tests.
       if (process.env.TEST === "unit") {
         nock(`${process.env.BITCOINCOM_BASEURL}`)
-          .get(/addr\/mgps7qxk2Z5ma4mXsviznnet8wx4VvMPFz.*/)
+          .get(`/addr/mgps7qxk2Z5ma4mXsviznnet8wx4VvMPFz?from=0&to=1000`)
```

Comment:
Because the regex ends in `*/` I was not able to properly isolate code in order to debug it with block comments (`/* */`). So I reverted all the regex to explicit format.